src/material/core/common-behaviors/color.spec.ts

```diff
@@ -1,10 +1,23 @@
+import {TestBed} from '@angular/core/testing';
 import {mixinColor} from './color';
-import {ElementRef} from '@angular/core';
+import {Component, ElementRef, Inject} from '@angular/core';
 
 describe('MixinColor', () => {
   it('should augment an existing class with a color property', () => {
-    const classWithColor = mixinColor(TestClass);
-    const instance = new classWithColor();
+    @Component({})
+    class TestComponent extends mixinColor(BaseComponent) {
```

Comment:
I don't think we should do this. I'm in the process of moving classes away from the `mixinColor` which also switches the theming to use host bindings.

Reply:
Even better! :) FWIW, I'm just experimenting with migration approaches.

Reply:
We don't use the `mixinColor` anywhere as of https://github.com/angular/components/pull/28048.

Reply:
Awesome!